Document/Document.ipynb

```diff
@@ -2516,6 +2516,15 @@
     "* run `help(Compare)` for more information"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<ul>\n",
+    "    <li><span style=\"color:red;\">Notice </span> :  `weight` renamed to `class_weight` in <span style=\"color:red;\">version 3.3</span> </li>\n",
```

Comment:
I think it's better to say: 

`weight` has been renamed to `class_weight` from version 3.3

Reply:
Good point!
We have also some similar sections, I think it would be better if we change them in another PR.